### PR TITLE
Sara/mkt 193 document rbac feature

### PR DESCRIPTION
**Pull request by s-santillan**
### Security

- [x] Change has no security implications (otherwise, ping the security team)

### CODE REVIEW

File: docs/semgrep-app/rbac.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+append_help_link: true
+title: Role-based access control
+description: "This page informs users of role-based access control in Semgrep App and teaches users about roles and role management."
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Controlling access through roles
+
+![Screenshot of x ](../img/rbac-overview.png)<br />
```
Comment (pabloest) [inline docs/semgrep-app/rbac.md:12]:
```suggestion
![Screenshot of role-based access control ](../img/rbac-overview.png)<br />
```

File: docs/semgrep-app/rbac.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Access control in Semgrep App determines the resources and features that are available to users based on their role. This **role-based access control (RBAC)** feature is available for organizations on Team/Enterprise tiers.
+
+## Setting up RBAC
+
+Semgrep App divides users into two roles:
```
Comment (pabloest) [inline docs/semgrep-app/rbac.md:18]: I just thought that “users” may be confused with “developer users” (meaning how we charge for the number of seats in Semgrep App). So maybe rephrase this to something like “Semgrep App divides users who log in into two roles:”

@raghavjain3 Do you think there’s potential for confusion with “developer users” here?

Comment (raghavjain3) [reply docs/semgrep-app/rbac.md:18]: I think the current wording is fine and implies that it's app users but also open to change to your suggestion of "Semgrep App divides users who log in into two roles" if that is more clear!

**Review by raghavjain3 — APPROVED** [review]
Approving but will wait for the +1 from @pabloest as well

**Review by pabloest — APPROVED** [review]
